assessment.js

```diff
@@ -1,8 +1,8 @@
 //We'll learn about require later in the course
 var ask = require('./ask.js');
 
-var answer = 'not empty';
+var answer = 0;
 
-while (answer !== '' && answer !== 'SeCrEt') {
+while (answer <= 7 && answer >= 9) {
```

Comment:
Uhhh... yeah. I really didn't completely understand clearly what was needed here. Hence the supreme lack of logic in this case. I guess I misread the first line of the question completely (I thought we were evaluating for numbers only, rather than strings).